lib/checks/keyboard/main-is-top-level.js

```diff
@@ -0,0 +1,13 @@
+var landmarks = axe.commons.aria.getRolesByType('landmark');
+var parent = node.parentElement;
+while (parent){
+	var role = parent.getAttribute('role');
+	if (!role){
+		role = axe.commons.aria.implicitRole(parent);
```

Comment:
Wilco,

Do you want us to modify in the file lib/commons/aria/index for role application:
type: 'landmark' to type: 'structure'

role presentation is already has type: 'structure' so no change need for it.